src/sentry/static/sentry/app/utils/eventWaiter.tsx

```diff
@@ -109,14 +109,14 @@ class EventWaiter extends React.Component<Props, State> {
       firstIssue = issues.find((issue: Group) => issue.firstSeen === firstEvent) || true;
 
       recordAnalyticsFirstEvent({
-        key: 'first_event_recieved',
+        key: 'first_event_received',
         organization,
         project,
       });
     } else {
       firstIssue = firstEvent;
       recordAnalyticsFirstEvent({
-        key: 'first_transaction_recieved',
+        key: 'first_transaction_received',
```

Comment:
This cannot be changed until this is changed https://github.com/getsentry/reload/blob/67d393e63089a6044c499e3795fae8d169425e5a/reload_app/events.py#L667

Reply:
Also I just wouldn't change it since there's no way to join that data